kelly-o.md

```diff
@@ -0,0 +1,18 @@
+---
+layout: page
+title: KellyO
+---
+
+## What I do
+
+I manage the end user documentation for Kayako and help organize Write the Docs. :thumbsup:
+
+## My favorite things
+
+food & travel
+sci-fi :milky_way: & fantasy :sparkles:
```

Comment:
Right now, and Iain Banks book called Consider Phlebus, and the new season of Sense8 ❤️.